configs/icds-14lakh-dimagivalues-jan2020.yml

```diff
@@ -674,7 +677,7 @@ services:
     min_storage_per_node: 50GB
     process:
       cores_per_node: 8
-      ram_per_node: 64
+      ram_per_node: 16
```

Comment:
I think we'll want to increase this since we're migrating to a single node (instead of a cluster). See changes here: https://github.com/dimagi/commcare_resource_model/pull/31/commits/98f1fe5ae16f7b5fdc0a078c1074801a199b66fa